### ISSUE TITLE
Karazahn "Terestian Illhoof"

### ISSUE THREAD

**Issue by Inyu** [Helios-Empire/Helios-Tracker#216]
Kind of bug: npc/quest/spell/instance/ ( include a link to helios database please for link look below )
http://db.helios-empire.eu/?npc=15688
http://db.helios-empire.eu/?npc=17229
What bug do you encounter?:
this encounter still has a few bugs
-http://db.helios-empire.eu/?npc=17229 is there 2 times from the beginning and doesnt respawn
-there arnt any chains to attack during sacrifce and the afflicted player gets out after a few seconds
-the player with the highest threat got sacrificed
-during the encounter only a small amount of nonelite imps spawned around 4-5
How to reproduce your bug?:
do the boss
How it should work?:
-http://db.helios-empire.eu/?npc=17229 should be there only once and respawn
-there should be chains which have to be killed in order to release the player
-player with highest threat shouldnt be sacrifced
-during the boss fight there should gradually spawn red portals which summons small imps all the time
Operating System:
windows 10
- Please first search before making a new issue on our tracker.
- Please fill in above template to submit a new issue.
- Use always helios database links > http://db.helios-empire.eu to quests/npcs/spells etc.

**Comment by dikkedeur** [comment]
Can this be retested since last update?
